sandman/source/ncurses_context.cpp

```diff
@@ -0,0 +1,146 @@
+#include "ncurses_context.h"
+
+namespace Sandman::NCurses
+{
+
+	// This window is where messages from the logger are written to.
+	static WINDOW* s_LoggingWindow = nullptr;
+
+	WINDOW* GetLoggingWindow()
+	{
+		return s_LoggingWindow;
+	}
+
+	// This window is where user input is echoed to.
+	static WINDOW* s_InputWindow = nullptr;
+
+	WINDOW* GetInputWindow()
+	{
+		return s_InputWindow;
+	}
+
+	/*
+		Configure a window with "sensible" defaults.
+	*/
+	static void DefaultConfigureWindow(WINDOW* const p_Window)
```

Comment:
Naming it this way makes me think that this is the default function, but I suspect it's the only one, so it's clear if it's named ConfigureWindowDefaults